Exception when importing large SIARD 2 file

Description: importing a large (60GB) SIARD 2 file containing about 200 million records reaches what seems to be a SAX limitation. This stopped importing the current table and importing continued on the next table.

Steps required to reproduce the bug:
1. Have a SIARD2 file with more than 50 million records
2. convert it to something else

The version used was at https://github.com/keeps/db-preservation-toolkit/commit/73521d5f6c0b4419948bd9c91c98f74ffddc2946.

The dbptk-app.log.txt file contains private information. The error and log output near the error:

```
2016-10-12 19:14:44,943 [main] INFO  (c.d.m.s.i.c.SIARD2ContentImportStrategy) Progress: 4439000 rows of table dbo.File (59%)
2016-10-12 19:14:48,550 [main] INFO  (c.d.m.s.i.c.SIARD2ContentImportStrategy) Progress: 4443000 rows of table dbo.File (59%)
2016-10-12 19:14:50,441 [main] ERROR (c.d.m.s.i.c.SIARD2ContentImportStrategy) An error occurred converting table contents
com.databasepreservation.model.exception.ModuleException: A SAX error occurred during processing of XML table file at content/schema1/table15/table15.xml
    at com.databasepreservation.modules.siard.in.content.SIARD2ContentImportStrategy.importContent(SIARD2ContentImportStrategy.java:166)
    at com.databasepreservation.modules.siard.in.input.SIARDImportDefault.getDatabase(SIARDImportDefault.java:48)
    at com.databasepreservation.Main.run(Main.java:127)
    at com.databasepreservation.Main.internal_main(Main.java:80)
    at com.databasepreservation.Main.main(Main.java:62)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: org.xml.sax.SAXException: Fatal Error: line: 1; column: -1380856250; JAXP00010004: The accumulated size of entities is "50,000,001" that exceeded the "50,000,000" limit set by "FEATURE_SECURE_PROCESSING".
    at com.databasepreservation.modules.siard.in.content.SAXErrorHandler.fatalError(SAXErrorHandler.java:86)
    at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:180)
    at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:400)
    at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
    at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1465)
    at com.sun.org.apache.xerces.internal.impl.XMLScanner.checkEntityLimit(XMLScanner.java:1544)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.handleCharacter(XMLDocumentFragmentScannerImpl.java:1940)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEntityReference(XMLDocumentFragmentScannerImpl.java:1866)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:3058)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:606)
    at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:118)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:504)
    at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:848)
    at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:777)
    at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
    at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1213)
    at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:643)
    at com.databasepreservation.modules.siard.in.content.SIARD2ContentImportStrategy.importContent(SIARD2ContentImportStrategy.java:164)
    ... 9 common frames omitted
Caused by: org.xml.sax.SAXParseException: JAXP00010004: The accumulated size of entities is "50,000,001" that exceeded the "50,000,000" limit set by "FEATURE_SECURE_PROCESSING".
    at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:203)
    at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:177)
    ... 25 common frames omitted
2016-10-12 19:14:50,441 [main] INFO  (c.d.m.s.i.c.SIARD2ContentImportStrategy) Total of 4444670 row(s) processed
2016-10-12 19:14:50,441 [main] INFO  (c.d.m.s.i.c.SIARD2ContentImportStrategy) Obtained contents from table 'dbo.File'
2016-10-12 19:14:50,441 [main] INFO  (c.d.m.s.i.c.SIARD2ContentImportStrategy) Obtaining contents from table 'dbo.FileProperties'
```

--- a/dbptk-core/src/main/java/com/databasepreservation/Main.java
+++ b/dbptk-core/src/main/java/com/databasepreservation/Main.java
@@ -72,6 +72,10 @@ public class Main {
     logProgramStart();
     cli.logOperatingSystemInfo();
 
+    // avoid SAX processing limit of 50 million elements
+    System.setProperty("totalEntitySizeLimit", "0");
+    System.setProperty("jdk.xml.totalEntitySizeLimit", "0");
+
     int exitStatus = EXIT_CODE_GENERIC_ERROR;
     if (cli.usingUTF8()) {
       if (cli.shouldPrintHelp()) {

